core/src/consensus/stake/mod.rs

```diff
@@ -295,13 +295,16 @@ mod tests {
             genesis_stakes.insert(address1, 100);
             Stake::new(genesis_stakes, new_validator_set(Vec::new()))
         };
-        assert_eq!(Ok(()), stake.init(&mut state));
+        stake.init(&mut state).unwrap();
```

Comment:
Personally, I prefer `assert` to `unwrap` because it shows the exact line that failed without backtrace.

Reply:
`assert` overload makes me hard to focus, since `assert` draws a lot of attention visually for me. And if there are `unwrap()` that can possibly throw an error, there should be testcases for them.